pkg/e2e/harness_runner.go/harness_runner.go

```diff
@@ -0,0 +1,39 @@
+package harness_runner
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/onsi/ginkgo/v2"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+	"github.com/openshift/osde2e/pkg/common/util"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/common/prow"
+)
+
+var _ = ginkgo.Describe("[Suite: Tests] Test Harness", func() {
+	defer ginkgo.GinkgoRecover()
+	h := helper.New()
+
+	TimeoutInSeconds := float64(viper.GetFloat64(config.Tests.PollingTimeout))
+	util.GinkgoIt("should run until completion", func(ctx context.Context) {
+		log.Printf("Test harness timeout is %v", TimeoutInSeconds)
+		h.SetServiceAccount(ctx, viper.GetString(config.Tests.TestUser))
```

Comment:
@jbpratt glad you are looking at this PR. In the old addon implementation there is some unused features. 
They had implemented cleanup logic and this variable which could be used inside the test harness. 
Do we want to port all of those features though they aren't being used or leverage some of the newer ones being set up such as the impersonator funcs yall have created?